deploy/pivot_role/pivotRole.yaml

```diff
@@ -213,7 +213,8 @@ Resources:
             Action:
               - 'logs:PutLogEvents'
             Effect: Allow
-            Resource: '*'
+            Resource: 
+              - !Sub 'arn:aws:logs:*:${AWS::AccountId}:log-group:/${EnvironmentResourcePrefix}*'
```

Comment:
In CDK Pivot Role we have the resource as `arn:aws:logs:*:{self.account}:log-group:*` but here it is different

Reply:
I merged PutLogEvents with the "Logs" SID statement set and added a prefix for lambda function log groups under resources